src/lib/backend.ai-client-node.ts

```diff
@@ -3398,14 +3398,14 @@ class ScalingGroup {
    * @param {string} description - Scaling group description
    * @param {string} wsproxyAddress - wsproxy url (NEW in manager 21.09)
    */
-  async create(name, description = "", wsproxyAddress = null) {
+  async create(name, description = "", seheduler_opts = "{}", wsproxyAddress = null) {
     const input = {
       description: description,
       is_active: true,
       driver: "static",
       scheduler: "fifo",
       driver_opts: "{}",
-      scheduler_opts: "{}"
+      scheduler_opts: seheduler_opts
```

Comment:
This might be a typo error, perhaps?

Reply:
yes it was a typo error, so i revise it to 'scheduler_opts'